site/public/css/gradeable-notebook.css

```diff
@@ -144,6 +144,7 @@ img {
     @page {
         size: auto;
         margin: 0;
+        /* stylelint-disable-next-line at-rule-descriptor-no-unknown */
         break-inside: avoid;
```

Comment:
It's interesting that this ignore was necessary given that `break-inside` is widely available across browsers. (nothing actionable to do at this time, assuming the ignore is necessary)